examples/logs-passthrough-token/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3"
+services:
+  # Sample bash script producing logs.
+  logging:
+    image: alpine/curl:latest
+    container_name: logging
+    entrypoint: /curl.sh
+    depends_on:
+      - otelcollector
+    networks:
+      - test
+    volumes:
+      - ./curl.sh:/curl.sh
+  # OpenTelemetry Collector
+  otelcollector:
+    image:  quay.io/signalfx/splunk-otel-collector:0.86.0
```

Comment:
Do we want a specific tag here to avoid breaking changes, or should we use `latest` to avoid updating as time passes?